code/ship/afterburner.cpp

```diff
@@ -88,10 +85,19 @@ void afterburners_start(object *objp)
 		return;
 	}
 
-	if ( (objp->flags[Object::Object_Flags::Player_ship]) && (objp == Player_obj) ) {
-		unsigned int now;
-		now = timer_get_milliseconds();
+	unsigned int now = timer_get_milliseconds();
+
+	if (now - shipp->afterburner_last_end_time < (int)(sip->afterburner_min_time_to_restart * 1000.0f)) {
```

Comment:
This potentially changes AI behavior with afterburner.  Make sure you mention all possible behavior changes in your PR's.